exercises/J-array-get-set/exercises2.js

```diff
@@ -7,6 +7,8 @@
 */
 
 let numbers = [1, 2, 3]; // Don't change this array literal declaration
+numbers[numbers.length] = 4;
+numbers[0] = 1;
```

Comment:
Though this code works, the correct way to accomplish this exercise would be to use the prototypes method to add the new item to the end of the array using **.push**.